### DISCUSSION TITLE
CS2 support

### DISCUSSION THREAD

**Discussion by stece** [tltneon/lgsl#172] [Q&A] [answered]
I was wondering if this already supports CS2, and if not, if it's on the timetable? :)

**Comment by tltneon** [comment, marked as answer]
Hi @stece 
It was already answered https://github.com/tltneon/lgsl/issues/165#issuecomment-1784948219 :)
So LGSL do supports CS2 but can't get player names from server

**Reply by stece** [reply]
ah thanks! I missed that ^^

**Reply by tltneon** [reply]
@stece https://github.com/tltneon/lgsl/issues/165#issuecomment-1878259833
